Controller/AuthorizeController.php

```diff
@@ -26,8 +26,10 @@
  *
  * @author Chris Jones <leeked@gmail.com>
  */
-class AuthorizeController extends ContainerAware
+class AuthorizeController
 {
+    use ContainerAwareTrait;
```

Comment:
not good as the bundle supports PHP 5.3. Implement the interface without using the trait